packages/themes/src/components/datepicker.ts

```diff
@@ -0,0 +1,148 @@
+import { keyframes } from '@emotion/react';
+import { YoruStyleProperties } from '@yoru-ui/core';
+
+const opacity = keyframes`
+  0% {
+   opacity: 0;
+  }
+  50% {
+    opacity: 1;
+  }
+  100% {
+    opacity: 1;
+}
+`;
+
+const baseStyle: YoruStyleProperties = {
```

Comment:
and for the base font family please use **Arial** first, because our button already uses that font family

Reply:
To be changed in the next commit.

Reply:
that's good, but if we want to add a default font style do we can still override the default font style? @leonardmns12

Reply:
Yes, it can be overrided in the component levels (if needed).

Reply:
ok then